template.tpl

```diff
@@ -879,163 +885,162 @@ ___TEMPLATE_PARAMETERS___
+    ],
     "displayName": "Link Click Tracking Configuration",
+    "name": "linkTrackingConfig",
     "groupStyle": "ZIPPY_OPEN",
+    "type": "GROUP",
     "subParams": [
       {
-        "type": "SELECT",
-        "name": "linkTrackingType",
-        "displayName": "Tracking Type",
         "selectItems": [
           {
-            "value": "enableLinkClickTracking",
-            "displayValue": "Enable Automatic Link Click Tracking"
+            "displayValue": "Enable Automatic Link Click Tracking",
+            "value": "enableLinkClickTracking"
           },
           {
-            "value": "trackLinkClick",
-            "displayValue": "Track Link Click Event"
+            "displayValue": "Track Link Click Event",
+            "value": "trackLinkClick"
           }
         ],
-        "simpleValueType": true
+        "displayName": "Tracking Type",
+        "simpleValueType": true,
+        "name": "linkTrackingType",
+        "type": "SELECT"
       },
       {
-        "type": "GROUP",
+        "enablingConditions": [
+          {
+            "paramName": "linkTrackingType",
+            "type": "EQUALS",
+            "paramValue": "enableLinkClickTracking"
+          }
+        ],
         "name": "linkTrackingEnable",
+        "type": "GROUP",
         "subParams": [
           {
-            "type": "TEXT",
-            "name": "linkTrackingAllowlist",
+            "help": "Add a comma-separated list (or variable that returns an array) of HTML classes. If any of these classes is not on the HTML element when a link click event is triggered, the event will not be tracked. NOTE! This option overrides the Denylist Classes below.",
             "displayName": "Allowlist Classes",
             "simpleValueType": true,
-            "help": "Add a comma-separated list (or variable that returns an array) of HTML classes. If any of these classes is not on the HTML element when a link click event is triggered, the event will not be tracked. NOTE! This option overrides the Denylist Classes below.",
+            "name": "linkTrackingAllowlist",
+            "type": "TEXT",
             "valueHint": "class1,class2,class3"
           },
           {
-            "type": "TEXT",
-            "name": "linkTrackingDenylist",
+            "help": "Add a comma-separated list (or variable that returns an array) of HTML classes. If any of these classes is on the HTML element when a link click event is triggered, the event will not be tracked. NOTE! This option has no effect if Allowlist Classes is specified.",
             "displayName": "Denylist Classes",
             "simpleValueType": true,
-            "valueHint": "class1,class2,class3",
-            "help": "Add a comma-separated list (or variable that returns an array) of HTML classes. If any of these classes is on the HTML element when a link click event is triggered, the event will not be tracked. NOTE! This option has no effect if Allowlist Classes is specified.",
```

Comment:
Trivia: this trailing comma meant this was actually not valid JSON before this update to the export format. Wonder how that happened.